src/main/java/org/jboss/pnc/cleaner/builds/FailedBuildsCleaner.java

```diff
@@ -402,13 +402,13 @@ private void deleteGroupAndHostedRepo(String pkgKey, String repoName, IndyStores
             throws IndyClientException {
         StoreKey groupKey = new StoreKey(pkgKey, StoreType.group, repoName);
         if (stores.exists(groupKey)) {
-            logger.trace("{} group {} exists - deleting...", pkgKey, repoName);
+            logger.debug("{} group {} exists - deleting...", pkgKey, repoName);
```

Comment:
I changed this just to debug level as we're logging almost same thing one level higher. Also @thescouser89 please clarify if you want change log level of application to info at this point or I can keep it on debug for now, thank you.

Reply:
Sounds good to me!